package.json

```diff
@@ -69,6 +69,7 @@
     "@kbn/pm": "link:packages/kbn-pm",
     "@kbn/test-subj-selector": "link:packages/kbn-test-subj-selector",
     "@kbn/ui-framework": "link:packages/kbn-ui-framework",
+    "@types/elasticsearch": "^5.0.26",
```

Comment:
We already have this in devDeps: https://github.com/elastic/kibana/blob/apm/dt/package.json#L225

Reply:
Forgot the `--dev` flag :headdesk: